docs/rest_api.md

```diff
@@ -192,6 +200,14 @@ operation to be preformed. Each nic may have no more than one pending
 network operation; an attempt to queue a second action will result in an
 error.
 
+Authorization requirements:
+
+* Access to the project to which `<node>` is assigned.
+* If `<network>` is not public, access to the project named by the network's
+  `"access"` field is required.
```

Comment:
This feels redundant, since both need to be in the same project if the network isn't public.

Perhaps instead move the Note to "Possible errors", appending "or `<network>` must be public"?